src/testing/flow_solver_cases/naca0012.cpp

```diff
@@ -0,0 +1,95 @@
+#include "naca0012.h"
+#include <deal.II/base/function.h>
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_refinement.h>
+#include <deal.II/grid/grid_tools.h>
+#include <stdlib.h>
+#include <iostream>
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/grid/grid_tools.h>
+#include <deal.II/numerics/vector_tools.h>
+#include <deal.II/fe/fe_values.h>
+#include "physics/physics_factory.h"
+#include "dg/dg.h"
+#include <deal.II/base/table_handler.h>
+#include "mesh/grids/naca_airfoil_grid.hpp"
+#include "mesh/gmsh_reader.hpp"
+#include "functional/lift_drag.hpp"
+
+namespace PHiLiP {
+namespace Tests {
+//=========================================================
+// NACA0012
+//=========================================================
+template <int dim, int nstate>
+NACA0012<dim, nstate>::NACA0012(const PHiLiP::Parameters::AllParameters *const parameters_input)
+        : FlowSolverCaseBase<dim, nstate>(parameters_input)
+{
+}
+
+template <int dim, int nstate>
+void NACA0012<dim,nstate>::display_flow_solver_setup() const
+{
+    using PDE_enum = Parameters::AllParameters::PartialDifferentialEquation;
+    const PDE_enum pde_type = this->all_param.pde_type;
+    const double pi = atan(1.0) * 4.0;
+    std::string pde_string;
+    if (pde_type == PDE_enum::euler)                {pde_string = "euler";}
+    if (pde_type == PDE_enum::navier_stokes)        {pde_string = "navier_stokes";}
+    this->pcout << "- PDE Type: " << pde_string << std::endl;
+    this->pcout << "- Polynomial degree: " << this->all_param.grid_refinement_study_param.poly_degree << std::endl;
+    this->pcout << "- Courant-Friedrich-Lewy number: " << this->all_param.flow_solver_param.courant_friedrich_lewy_number << std::endl;
+    this->pcout << "- Freestream Reynolds number: " << this->all_param.navier_stokes_param.reynolds_number_inf << std::endl;
```

Comment:
This should only be displayed if `pde_type == PDE_enum::navier_stokes`

Reply:
Fixed in most recent commit.